frontend/components/appointment/dropConfirmationDialog.js

```diff
@@ -26,8 +26,8 @@ const ConfirmationDialog = (props) => {
                     <div>{'Previous start at: ' + ((updateEvent && updateEvent.schedule) ? updateEvent.schedule.start._date.toString().substr(4, 20) : '')}</div>
```

Comment:
While at it, these two lines seem to be prone to similar issues.

Reply:
Tui Calendar use another type of Date object, that may not have `_date` property
it's called `TZDate`, not the same as plain js `Date` objects
also sometimes `start` and `end` can be strings instead of `Date` objects
that's why we got the issue for the two lines you changed
looking at these four lines, I think it's reasonable to say they are similar and can result in similar bugs